package.json

```diff
@@ -13,6 +13,7 @@
     "lerna": "^3.22.1"
   },
   "dependencies": {
+    "faker": "^5.4.0",
```

Comment:
This should be in `packages/api/package.json` so we can leverage this seed file in staging.